spec/configs/infinispan.xml

```diff
@@ -1,63 +1,65 @@
 <infinispan
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xsi:schemaLocation="urn:infinispan:config:10.0 http://www.infinispan.org/schemas/infinispan-config-10.0.xsd
-                            urn:infinispan:server:10.0 http://www.infinispan.org/schemas/infinispan-server-10.0.xsd"
-        xmlns="urn:infinispan:config:10.0"
-        xmlns:server="urn:infinispan:server:10.0">
+        xsi:schemaLocation="urn:infinispan:config:12.0 https://infinispan.org/schemas/infinispan-config-12.0.xsd
+                            urn:infinispan:server:12.0 https://infinispan.org/schemas/infinispan-server-12.0.xsd"
+        xmlns="urn:infinispan:config:12.0"
+        xmlns:server="urn:infinispan:server:12.0">
 
-    <cache-container default-cache="default">
-        <!--<global-state/>-->
-        <local-cache name="default" start="EAGER">
-            <locking acquire-timeout="30000" concurrency-level="1000" striping="false"/>
-            <transaction mode="NONE"/>
-            <encoding>
-                <key media-type="text/plain"/>
-                <value media-type="text/plain"/>
-            </encoding>
-        </local-cache>
-        <local-cache name="memcachedCache" start="EAGER">
-            <locking acquire-timeout="30000" concurrency-level="1000" striping="false"/>
-            <transaction mode="NONE"/>
-        </local-cache>
-        <local-cache name="namedCache" start="EAGER"/>
-    </cache-container>
+   <cache-container name="default" statistics="true" default-cache="default">
+      <local-cache name="default">
+         <locking acquire-timeout="30000" concurrency-level="1000" />
+         <encoding media-type="text/plain"/>
+      </local-cache>
+      <local-cache name="memcachedCache">
+         <locking acquire-timeout="30000" concurrency-level="1000" />
+         <encoding media-type="text/plain"/>
+      </local-cache>
+      <local-cache name="namedCache">
+         <encoding media-type="text/plain"/>
+      </local-cache>
+      <local-cache name="jsonCache">
+         <encoding media-type="text/plain"/>
```

Comment:
Shouldn't `jsonCache` have encoding `application/json`?

Reply:
no, it was text encode before as well